usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -732,6 +733,7 @@
 
         /* Delete paint sphere in hand when switching dominant hand */
         handChanged: function() {
+            Entities.callEntityMethod(_this.colorPaletteID, "createPaintSphere");
```

Comment:
does it matter that this call is before the `deleteEntity()` call below?

Reply:
It was supposed to be so to ensure it happens before this paint sphere gets deleted.

Reply:
OK - I _think_ that is safe.